cypress/e2e/user-journey.cy.ts

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const autosuggestData = require("../../wiremock/src/mappings/search/data/fbi/autosugggest.json");
+const searchResultData = require("../../wiremock/src/mappings/search/data/fbi/searchWithPagination.json");
+const availabilityLabelsData = require("../../wiremock/src/mappings/work/data/fbs/availability.json");
+
+describe("User journey", () => {
+  it("Shows search suggestions & redirects to search result page", () => {
+    cy.createMapping({
+      request: {
+        method: "POST",
+        url: "/opac/graphql",
+      },
+      response: {
+        headers: {
+          "Content-Type": "application/json",
```

Comment:
Please do the matching as suggested [here](https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/216/files#r1083912879).